rimage/rimage.h

```diff
@@ -133,16 +133,26 @@ struct image {
  */
 struct adsp {
 	const char *name;
+
```

Comment:
I think we could clean this up and use an array here (with a struct containing base and size) the index into the array could be the type enum (with BAR) ?

Reply:
Yes. I thought about this. Will try to implement in v2.

Reply:
New version of this patch is here:

https://github.com/dbaluta/sof/commit/7b82153727c2b484f291681ce9b80d27dd171822

Will send a PR as soon as #904 gets merged as it depends on it.